src/features/game/types/craftables.ts

```diff
@@ -1155,6 +1155,10 @@ export const COLLECTIBLES_DIMENSIONS: Record<CollectibleName, Dimensions> = {
   "Solar Flare Banner": { width: 1, height: 2 },
   "Human War Banner": { width: 1, height: 2 },
   "Goblin War Banner": { width: 1, height: 2 },
+
+  "Purple Trail": { width: 1, height: 1 },
+  Obie: { width: 1, height: 1 },
```

Comment:
Obie looks more like 1x2 in the renderings shown so far.

Reply:
Players right now are preferring 1x1 with some overflow as it looks nicer. It can be there decision if they want it to slightly block something above 👍